postgraphile/website/postgraphile/node-id.md

```diff
@@ -1,48 +1,50 @@
 ---
 layout: page
 path: /postgraphile/node-id/
-title: Globally Unique Object Identification ("nodeId" / "id")
+title: Globally Unique Object Identification ("id" / "nodeId")
 ---
 
 The [GraphQL Global Object Identification
 Specification](https://facebook.github.io/relay/graphql/objectidentification.htm)
-is one of the best practices in GraphQL, it gives clients a way to uniquely identify each object in the schema
-and to fetch these objects by their IDs.
-
-By default, PostGraphile implements this specification, assigning the unique
-identifier to every table with a primary key, but with a minor tweak - it uses
-the field name `nodeId` rather than the specified `id`. This change is to avoid
-clashing with the `id` field that is commonly the name of primary keys in
-database design. If you wish to call the Global Object Identifier field `id`
-instead (as is mandated by the specification), you can do so by overriding the
-`nodeIdFieldName` inflector, or you can use the `postgraphile/presets/relay`
-preset which does this (and a lot more) for you.
-
-If you choose to override the `nodeIdFieldName` inflector and you have database
-fields called `id` then you may want to override the `attribute` inflector too:
-
-```ts {7-9,12}
-const IdPlugin: GraphileConfig.Plugin = {
-  name: "IdPlugin",
-  version: "0.0.0",
-
-  inflection: {
-    replace: {
-      nodeIdFieldName() {
-        return "id";
-      },
-      attribute(previous, options, details) {
-        const name = previous!.call(this, details);
-        if (name === "id") return "rowId";
-        return name;
-      },
+is one of the best practices in GraphQL, it gives clients a way to uniquely identify
+each object in the schema and to fetch these objects by their IDs.
+
+By default, the `postgraphile/presets/amber` preset implements this specification.
+The amber preset assigns the unique identifier to every table with a primary key.
+The preset exposes the unique identifier as an attribute named `id`.
+
+It is common in database design to use the column name `id` for primary keys. For
+this reason, if there is an attribute named `id` that is already on the GraphQL type,
+the amber preset renames that attribute to `rowId`. If you want to use the amber
+preset but you do not like this behavior, you have several options:
+
+1. You can use the `postgraphile/presets/relay` preset which makes several changes
+    including removing `rowId` entirely.
+2. You can use a V4 preset that mimics the behavior and settings of Postgraphile V4
+    which used `nodeId` to represent the unique identifier.
+3. You can create your plugin similar to the following:
+
+```ts
+const RevertToNodeIdPlugin: GraphileConfig.Plugin = {
+    name: 'RevertToNodeIdPlugin',
+    version: '1.0.0',
+    inflection: {
+        replace: {
+            nodeIdFieldName: (): string => 'nodeId',
+            attribute: (previous: any, options: any, details: any) => {
```

Comment:
Theoretically these should be automatically typed for you, you should not need to declare them as any!

```suggestion
            attribute(previous, options, details) {
```